src/elements/yes-no-element.js

```diff
@@ -1,15 +1,28 @@
 import TextualElement from './textual-element';
+import Choice from './choice';
 
 export default class YesNoElement extends TextualElement {
   constructor(parent, attributes) {
     super(parent, attributes);
 
-    this.positiveChoice = attributes.positive;
+    this.positiveChoice = new Choice(attributes.positive);
```

Comment:
Using actual Choice objects so that we can use the toJSON method on the Choice when serializing. This _shouldn't_ break anything since the Choice object has the same attributes as the original JSON.

Reply:
This looks fine to me. Just need to test what gets written/read to the DB.